dev_guide/integrating_external_services.adoc

```diff
@@ -53,12 +53,20 @@ ignore the service and allow you to specify endpoints manually:
 ----
 {
   "kind": "Service",
-  "apiVersion": "v1beta3",
+  "apiVersion": "v1",
   "metadata": {
     "name": "example-mysql-service"
   },
-  "spec": { <1>
```

Comment:
yeah that's what it was supposed to link to and if you read the paragraph above (which says "leave the selector field unset") then it makes sense i guess.